index.html

```diff
@@ -1,58 +1,161 @@
+    <link rel="stylesheet" href="style.css" />
 
-<head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link rel="stylesheet" href="style.css" />
+    <title>Yousef Hesham</title>
+  </head>
 
-  <title>Yousef Hesham</title>
-</head>
-
-<body>
-  <div class="wrapper">
-
-    <header class="head-toolbar">
-      <a class="toolbar-title" href="#">
-        Welcome
-      </a>
-      <nav class="navbar">
-      
+  <body>
+    <div class="wrapper">
+      <header class="head-toolbar">
+        <a class="toolbar-title" href="#"> Welcome </a>
+        <nav class="navbar">
           <a href="#"> <img src="assets/images/hamburger.svg" alt="" /> </a>
-        
-      </nav>
-    </header>
+        </nav>
+      </header>
+
+      <section class="headline">
+        <h2 class="welcome-message">
+          Hey there.I’m Yousef <br />
+          <span>I’m a software developer </span>
+        </h2>
+        <p class="description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="social">
+          <ul class="social-icons">
+            <li class="items">
+              <a href="#"> <img src="assets/images/twitter.svg" alt="" /></a>
+            </li>
+            <li class="items">
+              <a href="#">
+                <img src="assets/images/Linkedinicon.svg" alt="" />
+              </a>
+            </li>
+            <li class="items">
+              <a href="#"> <img src="assets/images/medium.svg" alt="" /> </a>
+            </li>
+            <li class="items">
+              <a href="#"> <img src="assets/images/github.svg" alt="" /> </a>
+            </li>
+            <li class="items">
+              <a href="#"> <img src="assets/images/angellist.svg" alt="" /> </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="recent-works">
+        <img class="disabled-button" src="assets/images/Disabled.svg" alt="" />
+        <h2 class="work-title">My Recent Works</h2>
+        <hr class="horizontal-line" />
+        <article class="card">
+          <div class="project-sample">
+            <h4 class="project-title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project-lang-list">
+              <li>
+                <p class="lang">Ruby on rails</p>
+              </li>
+              <li>
+                <p class="lang">css</p>
+              </li>
+              <li><p class="lang">javScript</p></li>
+              <li>
+                <p class="lang">html</p>
+              </li>
+            </ul>
+            <a href="#" class="button"> See Projects </a>
+          </div>
+        </article>
+        <article class="card">
```

Comment:
According to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468), the project is five (5) and you have three implemented in your design. Kindly implement the remaining two to conform with the design.